content/pages/pension/eligibility.md

```diff
@@ -47,8 +42,9 @@ If you’re a surviving spouse or a child of a deceased Veteran with wartime ser
 
 </div>
 
-<div id="react-applicationStatus" class="static-page-widget">
-  <a class="usa-button-primary va-button-primary" href="/pension/application/527EZ">Apply for a Veterans Pension</a>
-</div>
+### Ready to apply? 
+
+You can apply by mail using an Application for Pension (VA Form 21P-527EZ). <br> 
```

Comment:
Would it be of interest to consider adding "or in person" in the future, and not necessarily in this PR?

Reply:
Thank you! We were waffling on this point. Since you noticed it too, we're adding "in person" and "with the help of a trained professional." Many thanks.